### ISSUE TITLE
How to add link to files?

### ISSUE THREAD

**Issue by Shiffalikansal** [eu81273/angular.treeview#63]
When i click on the file in treeview  then i want that some other html page should open. So i want to add link  to each file(childnode) in json itself and the use it in html page?

**Comment by awadhesh22791** [comment]
Yes it is possible to do this by editing tree view library and add anchor tag with href as follows
<a href="{{getLinkToRedirect()}}">
Define function as follows
$scope.getLinkToRedirect=function(){
return url;
}

This will place required link on href.
